doc/source/performance/threadpool.rst

```diff
@@ -4,19 +4,17 @@
  Thread Pool
 =============
 
+Servers continually execute queries from multiple clients. In *MySQL*, for each connection query, the server creates a thread, processes the query, and then destroys the thread. This method can have disadvantages because the server must consume resources along with the consuming the resources needed to process the query. Therefore, when the number of connections grows the server performance drops. Too many active threads impact performance because of context switching, and thread contention.
```

Comment:
"This method can have disadvantages because" -> I would remove this part

Reply:
along with the consuming the resources -> along with consuming the resources

Reply:
Therefore, when the number of connections grows the server performance drops. -> remove the comma after "Therefore" and add a comma after "grows"

Reply:
Keeping the disadvantages
fixed
fixed.